XamlStyler.Core/Options/DefaultSettings.json

```diff
@@ -15,8 +15,8 @@
     "xmlns:*",
     "x:Key, Key, x:Name, Name, x:Uid, Uid, Title",
     "Grid.Row, Grid.RowSpan, Grid.Column, Grid.ColumnSpan, Canvas.Left, Canvas.Top, Canvas.Right, Canvas.Bottom",
-    "Width, Height, MinWidth, MinHeight, MaxWidth, MaxHeight",
-    "Margin, Padding, HorizontalAlignment, VerticalAlignment, HorizontalContentAlignment, VerticalContentAlignment, Panel.ZIndex",
+    "Width, Height, WidthRequest, HeightRequest, MinWidth, MinHeight, MaxWidth, MaxHeight",
```

Comment:
I'd suggest to also add `MinimumWidthRequest, MinimumHeightRequest` after `MinWidth, MinHeight` since we have these properties in `VisualElement` which is base class for all UI elements in Xamarin.Forms

Reply:
Done